### PR TITLE
Fonts

### PR DESCRIPTION
**Pull request by Redth**

### CODE REVIEW

File: Resizetizer.NT/Resizetizer.NT.targets [hunk 1]
```diff
@@ -54,6 +68,12 @@
 			$(ResizetizeAfterTargets);
 			ResizetizeCollectItems;
 		</ResizetizeAfterTargets>
+
+		<ProcessSharedFontsAfterTargets>
+			$(ProcessSharedFontsAfterTargets);
+			_CompileCoreMLModels;
```
Comment (mattleibow) [inline Resizetizer.NT/Resizetizer.NT.targets:74]: 😬  Is this the best place?

File: Resizetizer.NT/Resizetizer.NT.targets [hunk 2]
```diff
@@ -159,8 +201,110 @@
...
+	<Target Name="ProcessSharedFonts"
+		Inputs="@(SharedFont);$(_SharedFontInputsFile)"
+		Outputs="$(_SharedFontStampFile)"
+		AfterTargets="$(ProcessSharedFontsAfterTargets)"
+		BeforeTargets="$(ProcessSharedFontsBeforeTargets)"
+		DependsOnTargets="$(ProcessSharedFontsDependsOnTargets)">
+
+		<DetectInvalidResourceOutputFilenamesTask
+			Items="@(SharedFont)"
+			ErrorMessage="One or more invalid file names were detected.  File names must be lowercase, start and end with a letter character, and contain only alphanumeric characters or underscores: &#13;&#10;">
```
Comment (mattleibow) [inline Resizetizer.NT/Resizetizer.NT.targets:221]: Maybe the newline should be in the task? 

Also, the two messages look the same? Maybe it is fine to have the default as this in the task and we can override in the targets if need be?

File: Resizetizer.NT/Resizetizer.NT.targets [hunk 3]
```diff
@@ -159,8 +201,110 @@
...
+			SkipUnchangedFiles="true" />
+
+
+		<ItemGroup>
+			<_SharedFontCopied Include="$(IntermediateOutputPath)sp\fonts\*" />
+		</ItemGroup>
+
+		<ItemGroup Condition="'$(_ResizetizerIsiOSApp)' == 'True' And '@(_SharedFontCopied)' != ''">
+
+			<!-- iOS Expects images in this group -->
```
Comment (mattleibow) [inline Resizetizer.NT/Resizetizer.NT.targets:237]:
```suggestion
			<!-- iOS Expects fonts to be in this group -->
```

File: Resizetizer.NT/Resizetizer.NT.targets [hunk 4]
```diff
@@ -159,8 +201,110 @@
...
+			</ContentWithTargetPath>
+
+		</ItemGroup>
+
+		<!-- iOS Only -->
+		<!-- If on Windows, using build host, copy the files over to build server host too -->
+		<CopyFilesToBuildServer
+			Condition="'$(BuildSessionId)' != '' And '$(_ResizetizerIsiOSApp)' == 'True' And '$(IsMacEnabled)'=='true'"
+			SessionId="$(BuildSessionId)"
+			Files="@(_SharedFontBundleResource)" />
```
Comment (mattleibow) [inline Resizetizer.NT/Resizetizer.NT.targets:297]: Will this copy the partial manifest as well? It doesn't seem to be in that group.

**Review by mattleibow — COMMENTED** [review]
Got a few Q's.

Also we should go through later and make the targets file consistent. Some things are in properties, others are just added - like the paths for the stamps.  But not essential, just nitpicky.

File: Resizetizer.NT/Resizetizer.NT.targets (hunk 1 above)
Comment (Redth) [reply Resizetizer.NT/Resizetizer.NT.targets:74]: For now, eventually there will be a better hook as per https://github.com/xamarin/xamarin-macios/issues/10646

File: Resizetizer.NT/Resizetizer.NT.targets (hunk 4 above)
Comment (Redth) [reply Resizetizer.NT/Resizetizer.NT.targets]: Good catch, fixed now.

File: Resizetizer.NT/Resizetizer.NT.targets (hunk 2 above)
Comment (Redth) [reply Resizetizer.NT/Resizetizer.NT.targets]: Was thinking more for translations, keeping the messages in the .targets...  I refactored into a shared property at least now.

**Review by mattleibow — APPROVED** [review]
lookin' good!

**Pull request merged**